src/tools/kata-ctl/README.md

```diff
@@ -0,0 +1,49 @@
+# Kata Containers control tool
+
+## Overview
+
+The `kata-ctl` tool is a rust rewrite of the
+[`kata-runtime`](../../runtime/cmd/kata-runtime)
+[utility program](../../../docs/design/architecture/README.md#utility-program).
+
+The program provides a number of utility commands for:
+
+- Using advanced Kata Containers features.
+- Problem determination and debugging.
+
+## Audience and environment
+
+Users and administrators.
+
+## Build the tool
+
+```bash
+$ make
+```
+
+## Install the tool
+
+```bash
+$ make install
```

Comment:
Don't need `sudo`? Normally `$` means a non-root user.

Reply:
It currently installs to the cargo bin dir, so this is correct. However, I agree we should start thinking about a proper install so I've raised https://github.com/kata-containers/kata-containers/issues/5403.